nova_plugin/keypair.py

```diff
@@ -148,7 +148,7 @@ def validate_path_owner(path):
 
     if is_external_resource_not_conditionally_created(ctx):
         if pk_exists:
-            if platform.system() == 'Linux':
+            if platform.system() in ('Linux', 'Darwin'):
```

Comment:
This fixes a test for this validation. Perhaps it should be `if os.name == 'posix'` instead?